src/main/java/com/cburch/logisim/file/LibraryManager.java

```diff
@@ -17,12 +17,8 @@
 import java.io.File;
 import java.io.IOException;
 import java.lang.ref.WeakReference;
-import java.util.ArrayList;
-import java.util.Collection;
-import java.util.HashMap;
-import java.util.HashSet;
-import java.util.Set;
-import java.util.WeakHashMap;
+import java.util.*;
```

Comment:
We agreed on using only explicit imports of the used classes.
Please change the broad `util.*` back to the explicit imports.